content/pages/health-care/about-va-health-care/assisted-living-and-home-health-care.md

```diff
@@ -65,7 +65,7 @@ You may be able to use one or more of these services if:
 
 We may also consider other factors, like your service-connected disability status or insurance coverage.
 
-To find out how to access these services, contact your VA social worker. Or, call our toll-free hotline at 877-222-VETS (<a href="tel:+1-877-222-8387">877-222-8387</a>), Monday through Friday, 8:00 a.m. to 8:00 p.m. (ET). 
+To find out how to access these services, contact your VA social worker. Or, call our toll-free hotline at 877-222-VETS (<a href="tel:+1-877-222-8387">1-877-222-8387</a>), Monday through Friday, 8:00 a.m. to 8:00 p.m. (ET). 
```

Comment:
Should there be a +1 before 877-222-VETS?

Reply:
@bethpotts ^^

Reply:
Yes, please!

Reply:
For all the examples Sophia found of vanity phone numbers, please do include the 1-